winpr/libwinpr/sspi/Negotiate/negotiate.c

```diff
@@ -72,45 +99,171 @@ const SecPkgInfoW NEGOTIATE_SecPkgInfoW = {
 	NEGOTIATE_SecPkgInfoW_Comment /* Comment */
 };
 
-static void negotiate_SetSubPackage(NEGOTIATE_CONTEXT* context, const TCHAR* name)
+static const sspi_gss_OID_desc spnego_OID = { 6, "\x2b\x06\x01\x05\x05\x02" };
```

Comment:
Not a big deal, but IIRC these are already defined in libkrb5.
Another remark for kerberos there's 2 OID: the official one and the microsoft's one (the official one with endianess problem in the middle of the OID). You can check in libkrb5's code to see how they adapt to that.

Reply:
Couldn't find them in the krb5 headers, but either way I think it's a bit much to include them just for that (especially since they may not be available in every build). Although it might be better to define them in one of the sspi headers
Regarding the old microsoft OID, I'll try to put in support for it